Make commands v2 the default

taldcroft opened the pull request:
## Description

This makes commands v2 the default and thus v1 is now the backup.

## Interface impacts
<!-- API changes, file format updates, coordination of changes with the community. -->
There are API changes that have been widely discussed in informational forums. In particular getting commands may try using network resources by default.

The new version of commands archive files generated with PR #248 should be installed into `$SKA/data/kadi` on HEAD when kadi commands v2 is promoted via a new ska3-flight release. Those files are in `/home/aldcroft/tmp/kadi` on HEAD.

## Testing
<!-- If relevant describe any special setup for testing. -->

### Unit tests
<!-- At least one of these must be checked if unit tests exist. DELETE the unchecked/untested options. -->
- [x] Mac

Independent check of unit tests by Jean
- [x] Linux

### Functional tests
<!-- Describe and document results of any functional tests, otherwise leave the text below -->
No functional testing.

jeanconn commented:
Did the docs lose the a breadcrumb to set to v1 with commands.conf.commands_version = "1"?

taldcroft commented:
@jeanconn - I rebased to fix the merge conflicts and re-ran unit tests with no failures. I also added a docs file `docs/commands_v1.rst` which I had previously forgotten to add. This file contains the requested breadcrumbs.
